{"0.1":{"info":{"author":"Fl√°vio Juvenal","author_email":"flaviojuvenal@gmail.com","bugtrack_url":null,"classifiers":["Development Status :: 4 - Beta","License :: OSI Approved :: MIT License","Topic :: Utilities"],"description_content_type":null,"docs_url":null,"download_url":"UNKNOWN","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/fjsj/retry_on_exceptions","keywords":"decorator decorators retry exception","license":"MIT License","maintainer":null,"maintainer_email":null,"name":"retry_on_exceptions","package_url":"https://pypi.org/project/retry_on_exceptions/","platform":"UNKNOWN","project_url":"https://pypi.org/project/retry_on_exceptions/","project_urls":{"Download":"UNKNOWN","Homepage":"https://github.com/fjsj/retry_on_exceptions"},"provides_extra":null,"release_url":"https://pypi.org/project/retry_on_exceptions/0.1/","requires_dist":null,"requires_python":null,"summary":"Decorator for retrying a function N times in case some specified exception occurs.","version":"0.1","yanked":false,"yanked_reason":null},"last_serial":798927,"urls":[{"comment_text":"","digests":{"blake2b_256":"8cbabbffc287baadfb3238085881b9cbdeb4b9b07135a13b2cb9757ec87c3316","md5":"64e32715bbe43b4fe4179d64f594c69c","sha256":"55dbb360d91917b7845ef144130585c4a62f39657908c4d76d6ea4149f5b192f"},"downloads":-1,"filename":"retry_on_exceptions-0.1.tar.gz","has_sig":false,"md5_digest":"64e32715bbe43b4fe4179d64f594c69c","packagetype":"sdist","python_version":"source","requires_python":null,"size":3281,"upload_time":"2012-09-28T20:25:41","upload_time_iso_8601":"2012-09-28T20:25:41.323494Z","url":"https://files.pythonhosted.org/packages/8c/ba/bbffc287baadfb3238085881b9cbdeb4b9b07135a13b2cb9757ec87c3316/retry_on_exceptions-0.1.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}